cmd/create/cluster/cmd.go

```diff
@@ -1917,6 +1943,113 @@ func run(cmd *cobra.Command, _ []string) {
 		}
 	}
 
+	isSharedVPC := false
+	privateHostedZoneID := strings.Trim(args.privateHostedZoneID, " \t")
```

Comment:
Maybe we can add a validation that if `privateHostedZoneID` or `privateHostedZoneRoleARN` are provided all three new flags should be provided together.
We check this combination in the backend as well, but since it's simple logic, I think it's ok to validate in the CLI.
https://gitlab.cee.redhat.com/service/uhc-clusters-service/-/blob/master/cmd/clusters-service/servecmd/apiserver/validation_helpers.go#L2180
@ciaranRoche WDYT?

Reply:
Lets keep the validations to a min in the client, until everything is worked out. For MVP I think its okay to have this in the backend for now. Better to report late then wrong, and give us some freedom to change if needed and not tied to a release. 

@oriAdler lets set a call for tomorrow, and we capture these things in an improvements epic

Reply:
removed the changes for interactive mode